### PR TITLE
added identifier accessor for IdentifierNotFoundException

### PR DESCRIPTION
**Pull request by digitalkaoz**
EOM

### CODE REVIEW

File: src/Reflector/Exception/IdentifierNotFound.php [hunk 1]
```diff
@@ -6,12 +6,32 @@
 
 class IdentifierNotFound extends \RuntimeException
 {
+    /**
+     * @var Identifier
+     */
+    private $identifier;
+
+    public function __construct($message, $identifier)
```
Comment (asgrim) [inline src/Reflector/Exception/IdentifierNotFound.php]: please type hint `$identifier`

Comment (digitalkaoz) [inline src/Reflector/Exception/IdentifierNotFound.php]: done

**Comment by Ocramius** [comment]
What's the exact use-case for this, in a consumer?

**Comment by digitalkaoz** [comment]
yes i use this to get the "unloadable" class back from the Exception to do further things. parsing it from the message felt a bit ugly

**Comment by asgrim** [comment]
@Ocramius I have no problem with this, I'm gonna merge

**Pull request merged**